extras/Waydroid-Toolbox.sh

```diff
@@ -34,7 +34,7 @@ echo -e "$PASSWORD\n" | sudo -S ls &> /dev/null
 if [ $? -ne 0 ]
 then
 	echo sudo password is wrong! | \
-		zenity --text-info --title "Clover Toolbox" --width 400 --height 200
+		zenity --text-info --title "Waydroid Toolbox" --width 400 --height 200
```

Comment:
nice catch

Reply:
This was split out into https://github.com/ryanrudolfoba/SteamOS-Waydroid-Installer/commit/94ed62e3ef882f37c287fa57e48ef15cc27ee945